url/url-setters.any.js

```diff
@@ -6,7 +7,7 @@
 
 // Keep this file in sync with url-setters-a-area.window.js.
 
-promise_test(() => fetch("resources/setters_tests.json").then(res => res.json()).then(runURLSettersTests), "Loading data…");
+promise_test(() => fetch_json("resources/setters_tests.json").then(runURLSettersTests), "Loading data…");
```

Comment:
Where does fetch_json come from?

Reply:
I added it in [RFC 177](https://github.com/web-platform-tests/rfcs/blob/master/rfcs/fetch_json_et_al.md#add-fetch_json-test-harness-api) earlier this year.